suite/runsuite.cmake

```diff
@@ -319,30 +319,28 @@ if (UNIX AND ARCH_IS_X86)
   set(ENV{CXXFLAGS} "")
   set(prev_run_tests ${run_tests})
   set(run_tests OFF) # build tests but don't run them
-  if (NOT cross_android_only)
```

Comment:
OK you're removing this b/c the "optional_cross_compile" will ignore the expected error b/c the Android Travis setup has no arm compiler